instructor/common.py

```diff
@@ -694,6 +698,8 @@ def western_blot_band_size(request):
             for antibody in antibodies:
                 for index, field in enumerate(weights_by_type):
                     weight_str = getattr(antibody, field)
+                    error = True if bool(re.search(r'[a-zA-Z-]+', weight_str)) else error
```

Comment:
`bool()` isn't necessary here since the `if` expression will do that evaluation for you